src/V2/ApplicationV2/OmpOpcUaClient.cs

```diff
@@ -266,7 +317,7 @@ public async Task CloseSessionAsync(string endpointUrl, CancellationToken cancel
         }
         #endregion
 
-        protected Task<IOpcUaSession> GetSession(string endpointUrl, CancellationToken cancellationToken)
-            => sessionPoolStateManager.GetSessionAsync(endpointUrl, cancellationToken);
+        protected async Task<IOpcUaSession> GetSessionAsync(string endpointUrl, CancellationToken cancellationToken)
+            => await sessionPoolStateManager.GetSessionAsync(endpointUrl, cancellationToken);
```

Comment:
Can we remove this await (and async) keyword. It serves no purpose and will only incur an unnecessary state machine

Reply:
Done